src/rust/core/server/src/threads/admin.rs

```diff
@@ -27,32 +27,53 @@ use std::net::SocketAddr;
 use std::sync::Arc;
 use tiny_http::{Method, Request, Response};
 
-static_metrics! {
-    static ADMIN_REQUEST_PARSE: Counter;
-    static ADMIN_RESPONSE_COMPOSE: Counter;
-    static ADMIN_EVENT_ERROR: Counter;
-    static ADMIN_EVENT_WRITE: Counter;
-    static ADMIN_EVENT_READ: Counter;
-    static ADMIN_EVENT_LOOP: Counter;
-    static ADMIN_EVENT_TOTAL: Counter;
-
-    static RU_UTIME: Counter;
-    static RU_STIME: Counter;
-    static RU_MAXRSS: Gauge;
-    static RU_IXRSS: Gauge;
-    static RU_IDRSS: Gauge;
-    static RU_ISRSS: Gauge;
-    static RU_MINFLT: Counter;
-    static RU_MAJFLT: Counter;
-    static RU_NSWAP: Counter;
-    static RU_INBLOCK: Counter;
-    static RU_OUBLOCK: Counter;
-    static RU_MSGSND: Counter;
-    static RU_MSGRCV: Counter;
-    static RU_NSIGNALS: Counter;
-    static RU_NVCSW: Counter;
-    static RU_NIVCSW: Counter;
-}
+#[metric(name="admin_request_parse", crate=common::metrics)]
+static ADMIN_REQUEST_PARSE: Counter = Counter::new();
+#[metric(name="admin_response_compose", crate=common::metrics)]
+static ADMIN_RESPONSE_COMPOSE: Counter = Counter::new();
+#[metric(name="admin_event_error", crate=common::metrics)]
+static ADMIN_EVENT_ERROR: Counter = Counter::new();
+#[metric(name="admin_event_write", crate=common::metrics)]
+static ADMIN_EVENT_WRITE: Counter = Counter::new();
+#[metric(name="admin_event_read", crate=common::metrics)]
+static ADMIN_EVENT_READ: Counter = Counter::new();
+#[metric(name="admin_event_loop", crate=common::metrics)]
+static ADMIN_EVENT_LOOP: Counter = Counter::new();
+#[metric(name="admin_event_total", crate=common::metrics)]
+static ADMIN_EVENT_TOTAL: Counter = Counter::new();
+
+#[metric(name="ru_utime", crate=common::metrics)]
+static RU_UTIME: Counter = Counter::new();
+#[metric(name="ru_stime", crate=common::metrics)]
+static RU_STIME: Counter = Counter::new();
+#[metric(name="ru_maxrss", crate=common::metrics)]
```

Comment:
@brayniac not sure if there's a way to avoid this really repetitive `crate = common::metrics` stuff while re-exporting the metrics stuff from `common`